static/js/components/PersonalTab.js

```diff
@@ -58,16 +60,29 @@ class PersonalTab extends React.Component {
     setProgram(selected);
   }
 
+  getProgramId = (selectedProgram: Program, currentProgramEnrollment: ProgramEnrollment) => {
```

Comment:
also, i think it makes sense to get the values from `this.props` instead of passing them in as arguments here. 

```javascript
getSelectedProgramId = () : number|null => {
    const {
      ui: { selectedProgram },
      currentProgramEnrollment
    } = this.props;
    
    let programId = null;
    if (selectedProgram) {
      programId = selectedProgram.id;
    } else if(currentProgramEnrollment) {
      programId = currentProgramEnrollment.id;
    }
    return programId;
}
```